modules/cms/controllers/Index.php

```diff
@@ -59,6 +60,18 @@ public function __construct()
     {
         parent::__construct();
 
+        Event::listen('backend.form.extendFieldsBefore', function ($widget) {
```

Comment:
No need to add the event listener here, the form controller behavior provides a `formExtendFieldsBefore($widget)` method on the controller that you can override instead for the same end result.